### DISCUSSION TITLE
Please if the developer can. Support mv2 until the summer of 2025 [2025H2]

### DISCUSSION THREAD

**Discussion by rtx2080tix** [uBlockOrigin/uBlock-issues#2987] [General]
asking about #338

Just like what the title says
And in the extension is removed from Chrome Web Store
You can just publish the release on **GitHub** so that users who want powerful and reliable can use powerful and reliable extension 

mv2 is **always more useful and powerful and reliable** than mv3
you can enable mv2 in 2024 using [ExtensionManifestV2Availability](https://chromeenterprise.google/policies/#ExtensionManifestV2Availability)
windows registry example

```
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\BraveSoftware\Brave]
"ExtensionManifestV2Availability"=dword:00000002

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Chromium]
"ExtensionManifestV2Availability"=dword:00000002

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Google\Chrome]
"ExtensionManifestV2Availability"=dword:00000002

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge]
"ExtensionManifestV2Availability"=dword:00000002

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Vivaldi]
"ExtensionManifestV2Availability"=dword:00000002

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\YandexBrowser]
"ExtensionManifestV2Availability"=dword:00000002


```
and linux user are nerd enough to apply the policy to any chromium based browser

And yes I know the policy is just a one-year delay
but I refused to use mv3 with useful and powerful extensions

i hope uBlock Origin mv2 version will be **supported until mv2 code is 100% removed from chromium source code**

**Comment by iam-py-test** [comment]
>  but I refused to use mv3 with useful and powerful extensions

If I recall correctly, Firefox will still support MV2 (maybe they changed their minds, though).
Thanks

**Reply by xlollomanx** [reply]
Actually FF support and will support both. For more info u can check their forum here: [https://blog.mozilla.org/addons/2022/05/18/manifest-v3-in-firefox-recap-next-steps/](https://blog.mozilla.org/addons/2022/05/18/manifest-v3-in-firefox-recap-next-steps/)

Honestly I think he's talking about the chromium version which will be removed from Chrome web store later next year.

**Reply by krystian3w** [reply]
The brave users may try to install unpacked raw XPI in Chrome "143". 


In part such a trick is done by Orion Browser on macOS they allow any addon shop, even though the engine driving it is Apple Webkit.
